StrainGauge/makefile

```
@@ -0,0 +1,6 @@
+all: StrainGauge
+
+StrainGauge.o: StrainGauge.cpp
+	g++ -Wall  -c StrainGauge.cpp -o StrainGauge.o
+StrainGauge: StrainGauge.o
+	g++ -Wall StrainGauge.o -o StrainGauge
```

Consider adding a "clean" stage to your makefile. If you do and then run "make clean", it will delete all of the object files in the folder so you don't have to do it manually.
```suggestion
	g++ -Wall StrainGauge.o -o StrainGauge
	
clean:
	rm -f *.o
```